hadoop-ozone/dist/src/main/license/jar-report.txt

```diff
@@ -210,45 +210,45 @@ share/ozone/lib/ozone-csi.jar
 share/ozone/lib/ozone-datanode.jar
 share/ozone/lib/ozone-filesystem-common.jar
 share/ozone/lib/ozone-filesystem-hadoop2.jar
+share/ozone/lib/ozone-filesystem-hadoop3-client.jar
 share/ozone/lib/ozone-filesystem-hadoop3.jar
 share/ozone/lib/ozone-filesystem.jar
-share/ozone/lib/ozone-filesystem-hadoop3-client.jar
 share/ozone/lib/ozone-insight.jar
 share/ozone/lib/ozone-interface-client.jar
 share/ozone/lib/ozone-interface-storage.jar
 share/ozone/lib/ozone-manager.jar
-share/ozone/lib/ozone-reconcodegen.jar
 share/ozone/lib/ozone-recon.jar
+share/ozone/lib/ozone-reconcodegen.jar
 share/ozone/lib/ozone-s3gateway.jar
 share/ozone/lib/ozone-tools.jar
 share/ozone/lib/perfmark-api.jar
 share/ozone/lib/picocli.jar
+share/ozone/lib/proto-google-common-protos.jar
+share/ozone/lib/protobuf-java-util.jar
 share/ozone/lib/protobuf-java.jar
 share/ozone/lib/protobuf-java.jar
-share/ozone/lib/protobuf-java-util.jar
-share/ozone/lib/proto-google-common-protos.jar
 share/ozone/lib/ranger-intg.jar
 share/ozone/lib/ranger-plugin-classloader.jar
 share/ozone/lib/ranger-plugins-audit.jar
 share/ozone/lib/ranger-plugins-common.jar
 share/ozone/lib/ranger-plugins-cred.jar
-share/ozone/lib/ratis-client.jar
-share/ozone/lib/ratis-common.jar
-share/ozone/lib/ratis-grpc.jar
-share/ozone/lib/ratis-metrics.jar
-share/ozone/lib/ratis-netty.jar
-share/ozone/lib/ratis-proto.jar
-share/ozone/lib/ratis-server-api.jar
-share/ozone/lib/ratis-server.jar
+share/ozone/lib/ratis-clientbdda-SNAPSHOT.jar
+share/ozone/lib/ratis-commonbdda-SNAPSHOT.jar
+share/ozone/lib/ratis-grpcbdda-SNAPSHOT.jar
+share/ozone/lib/ratis-metricsbdda-SNAPSHOT.jar
+share/ozone/lib/ratis-nettybdda-SNAPSHOT.jar
+share/ozone/lib/ratis-protobdda-SNAPSHOT.jar
+share/ozone/lib/ratis-server-apibdda-SNAPSHOT.jar
+share/ozone/lib/ratis-serverbdda-SNAPSHOT.jar
 share/ozone/lib/ratis-thirdparty-misc.jar
-share/ozone/lib/ratis-tools.jar
+share/ozone/lib/ratis-toolsbdda-SNAPSHOT.jar
```

Comment:
These look wrong, no dependency update is expected for version bump.  I found the bug in `update-jar-report.sh`, being fixed in #4034.  Once that's merged, 906fefbc8ff8c966c8922d45a44c0c217872f7d1 can be reverted from this PR.